addons/link_tracker/models/mail_render_mixin.py

```diff
@@ -3,6 +3,7 @@
 
 import re
 
+import markupsafe
```

Comment:
HumanLinter: markupsafe should be imported before re

Reply:
Nope, `re` is part of the standard library, markupsafe is not.

Reply:
Ah !

Reply:
But the other imports are a mess. PyCharm basically inserted MarkupSafe at the first existing non-stdlib import, but there are more stdlib imports afterwards. I should probably rewrite the entire list of imports here, but I would've expected pylint to  complain.